### PR TITLE
Slider browser unit tests

### PR DESCRIPTION
**Pull request by freddyaboulton**
## Description

Closes: #4679 

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests & Changelog

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

1. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`
  
1. Unless the pull request is labeled with the "no-changelog-update" label by a maintainer of the repo, all pull requests must update the changelog located in `CHANGELOG.md`:

Please add a brief summary of the change to the Upcoming Release section of the `CHANGELOG.md` file and include
a link to the PR (formatted in markdown) and a link to your github profile (if you like). For example, "* Added a cool new feature by `[@myusername](link-to-your-github-profile)` in `[PR 11111](https://github.com/gradio-app/gradio/pull/11111)`".

### CODE REVIEW

File: .config/basevite.config.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import { defineConfig } from "vite";
```
Comment (freddyaboulton) [inline .config/basevite.config.ts:1]: Will file an issue to refactor `vite.config.ts` to use the base config

File: .config/playwright-ct.config.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { defineConfig, devices } from "@playwright/experimental-ct-svelte";
+import config from "./basevite.config";
```
Comment (freddyaboulton) [inline .config/playwright-ct.config.ts:2]: This was mostly generated automatically by playwright except ctViteConfig

File: .config/playwright/index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (freddyaboulton) [inline .config/playwright/index.html:1]: Need these two playwright files in order to it to work

File: js/app/test/slider_release.spec.ts [hunk 4]
```diff
@@ -1,48 +0,0 @@
-import { Page, Locator } from "@playwright/test";
```
Comment (freddyaboulton) [inline js/app/test/slider_release.spec.ts:1]: No longer need to spin up a server for this test! We now test the change/release events in the Slider.component.spec.ts file

File: js/app/src/components/Slider/Slider.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+			maximum: 10,
+			label: "My Slider",
+			show_label: true,
+			step: 1,
+			mode: "dynamic",
+			loading_status: loading_status
+		},
+		on: {
+			change: () => ++change,
+			release: () => ++release
```
Comment (pngwn) [inline js/app/src/components/Slider/Slider.component.spec.ts:119]: Could we make these mocks or spies? I think we could just use `tinyspy` for these.

**Review by pngwn — APPROVED** [review]
One small comment but this look great to me. Thanks again @freddyaboulton!

**Comment by freddyaboulton** [comment]
Thank you for the review @pngwn !!

**Pull request merged**